editor-desktop/build.gradle

```diff
@@ -67,3 +85,148 @@ task distLegacyTest(type: Jar) {
 }
 
 dist.dependsOn classes
+
+
+import com.squareup.javapoet.FieldSpec
+import com.squareup.javapoet.JavaFile
+import com.squareup.javapoet.MethodSpec
+import com.squareup.javapoet.TypeSpec
+import javax.lang.model.element.Modifier
+import com.badlogic.gdx.Application
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.utils.SharedLibraryLoader
+import lombok.Getter
+
+def addStringField (TypeSpec.Builder builder, String fieldName, String fieldValue) {
+    FieldSpec fieldSpec = FieldSpec.builder(String.class, fieldName)
+            .addModifiers(Modifier.PRIVATE, Modifier.FINAL, Modifier.STATIC)
+            .addAnnotation(Getter.class)
+            .initializer('$S', fieldValue)
+            .build()
+
+    builder.addField(fieldSpec)
+}
+
+def addIntField (TypeSpec.Builder builder, String fieldName, int fieldValue) {
+    FieldSpec fieldSpec = FieldSpec.builder(int.class, fieldName)
+            .addModifiers(Modifier.PRIVATE, Modifier.FINAL, Modifier.STATIC)
+            .addAnnotation(Getter.class)
+            .initializer('$L', fieldValue)
+            .build()
+
+    builder.addField(fieldSpec)
+}
+
+task generateBuildConfig {
+
+    description 'Generate build config java file'
+
+    outputs.upToDateWhen { false }
+
+    doFirst {
+        String version = project.versionString
+        int versionCode = project.versionCode
```

Comment:
Don't need version code. version should come from project version in gradle.properties.